azkaban-common/build.gradle

```diff
@@ -72,6 +72,13 @@ dependencies {
   testRuntime('com.h2database:h2:1.4.193')
 }
 
+task copySqlFiles(type: Copy) {
+  from project(':azkaban-sql').file('src/sql')
```

Comment:
At the newly added test, I use in-memory db constructing AZ db tables, in order to fix out-dated tests. Those scripts have to be available in the module `azkaban-common`, so I made a copy.